src/ionization/ionization.jl

```diff
@@ -53,18 +59,37 @@ const σ_T = ustrip(natural(float(ThomsonCrossSection)))
 β(T_b) = α⁽²⁾(T_b) * (m_e * T_b / (2π))^(3/2) * exp(-ε₀_H / T_b)
 β⁽²⁾(T_b) = β(T_b) * exp(3ε₀_H / 4T_b)
 n₁ₛ(a, Xₑ, par) = (1 - Xₑ) * n_H(a, par)
-Λ_α(a, Xₑ, par) = H_a(a, par) * (3ε₀_H)^3 / ((8π)^2 * n₁ₛ(a, Xₑ, par))
+#Problem is here \/ since Lyα rate is given by redshifting out of line need H
+# Λ_α(a, Xₑ, par) = H_a(a, par) * (3ε₀_H)^3 / ((8π)^2 * n₁ₛ(a, Xₑ, par))
+Λ_α(a, Xₑ, par) = oldH_a(a, par) * (3ε₀_H)^3 / ((8π)^2 * n₁ₛ(a, Xₑ, par))
+new_Λ_α(a, Xₑ, par, ℋ_function) = ℋ_function(log(a)) * (3ε₀_H)^3 / ((8π)^2 * n₁ₛ(a, Xₑ, par))
```

Comment:
We should probably clean this up eventually, but this is fine for now to have two of these.

Reply:
I don't think any of these duplicates (or original functions) in this file are necessary if we are going to get rid of Peebles ionization history as an option (since now we have recfast). If we want to keep them then yes we should clean it up.